vzstorage-pd/provisioner-pod.yaml

```diff
@@ -0,0 +1,28 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: vz-provisioner-pod
+  namespace: kube-system
+spec:
+  hostNetwork: true
+  containers:
+  - name: virtuozzo-provisioner-pod
+    image: virtuozzo-storage:latest
+    imagePullPolicy: IfNotPresent
+    securityContext:
+      privileged: true
+    volumeMounts:
+      - name: internal
+        mountPath: /var/run/virtuozzo-provisioner
```

Comment:
/var/run is tmpfs, will be cleared on node restart.

BTW, what if we simply hardcode some identity value? I.e. keep it the same in every vzstorage-provisioner?